### ISSUE TITLE
[NoUnsortedConstructors] Fix removes parenthesis for constructor parameters

### ISSUE THREAD

**Issue by phenax** [fysiweb/elm-review-sorted#4]
```elm
20| type Testing
21|     = Zoobar (Maybe String)
22|     | Foobar (List String)
23|     | A
24| type Testing
24|     =A
24|     |Foobar List String
24|     |Zoobar Maybe String
```

**Expected** the type constructor parameters to retain the parenthesis and not break the build

**Comment by phenax** [comment]
Created an issue upstream with elm-syntax. https://github.com/stil4m/elm-syntax/issues/158

**Comment by phenax** [comment]
Maybe this project should consider migrating the writer responsibilities to [elm-syntax-dsl](https://package.elm-lang.org/packages/the-sett/elm-syntax-dsl/latest). It'll also resolve #3

@fysiweb, thoughts?

**Comment by phenax** [comment]
The issue has been fixed in v7.2.9 (https://github.com/stil4m/elm-syntax/pull/159). Closing this issue.
